front/src/applications/operationalStudies/views/Scenario/components/Timetable/Timetable.tsx

```diff
@@ -141,6 +145,46 @@ const Timetable = ({
   );
 
   const subCategories = useSubCategoryContext();
+  const previousSelectedTrainId = useRef<string | null>(null);
+
+  // Scroll to selected train element cliked in the conflict card
+  const scrollToSelectedTrain = useCallback(() => {
+    if (selectedTrainId) {
+      const selectedElement = document.querySelector(`[data-train-id="${selectedTrainId}"]`);
+      if (selectedElement) {
+        selectedElement.scrollIntoView({
+          behavior: 'smooth',
+          block: 'center',
+        });
+      }
+    }
+  }, [selectedTrainId]);
+
+  // Open paced train for occurrences when selected from conflict card
+  useEffect(() => {
+    if (selectedTrainId && selectedTrainId !== previousSelectedTrainId.current) {
+      const isOccurrence =
+        selectedTrainId.startsWith('indexedoccurrence_') ||
+        selectedTrainId.startsWith('exception_');
```

Comment:
We can use `isAddedExceptionId` and `isIndexedOccurrenceId` from traindId.ts
```suggestion
      const isOccurrence =
        isIndexedOccurrenceId(selectedTrainId) ||
        isAddedExceptionId(selectedTrainId)
```

Reply:
Good catch ! Done